aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,183 @@
+  <PageContent>
+
+    /* All contents within me will respect the 
+     * max-width, alignment, and padding specified 
+     * by Page's context. */
+     
+  </PageContent>
+  
+  /* Contents outside of PageContent are not bound by 
+   * Page's context. */
+
+</Page>
+```
+
+<br />
+
+In most cases, all content on a page can be wrapped in a [single PageContent component](#simple). 
+However, designs using [differing background colors](#full-width-background-color) to delineate page 
+sections may accomplish this by applying a PageContent per desired section.
+
+## Example use cases
+
+### Simple
+All content is placed within a single PageContent component.
+
+<Example
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/page/SinglePageContent.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/FilterControls/FilterControls.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/FilterControls/FiltersContext.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/PageHeader/PageHeader.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/data/mockData/trainingDataSets.json",
+  ]}
+   github="https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/components/page/SinglePageContent.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  width="100%" 
+  showResponsiveControls={['fullScreen']}
+  screenContainer={{ scale: 0.6 }}
+>
+  <SinglePageContent />
+</Example>
+
+### Full-width background color
+Use cases where an application needs a background to span the entire width of a page section 
+is achieved by setting `background={{ fill: "horizontal", ...additional properties }}` on 
+PageContent.
+
+This treatment is often seen on marketing sites where backgrounds help visually 
+block content sections. This might also be seen in a context such as an app-store catalog where 
+the hero section might have an image spanning.
+
+<Example
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/page/PageBackGroundExample.js",
+  ]}
+  github="https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/components/page/PageBackGroundExample.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  width="100%" 
+  showResponsiveControls={['fullScreen']}
+  screenContainer={{ scale: 0.57 }}
+>
+  <PageBackGroundExample />
+</Example>
+
+## Variants 
+
+### Wide
+"Wide" is Page's default `kind`. The vast majority of pages in HPE applications will 
+likely use Page's "wide" variant.
+
+See [page layouts](/templates/page-layouts#page-kinds) for specifications 
```

Comment:
Should we reference content layouts too?

So something like:
"See [Page Layouts](https://design-system.hpe.design/templates/page-layouts) for specifications and [Content Layouts](https://design-system.hpe.design/templates/content-layouts?q=content) for templates and guidance on responsive design."

Reply:
done